### DISCUSSION TITLE
Changes in shapefile made in Qfield are not saved?

### DISCUSSION THREAD

**Discussion by Iagree12** [opengisch/QField#2791] [Q&A]
Hi everybody,

I am new to QGIS and have the problem that the observations I entered into a shapefile in Qfield are only displayed in the app. Once I try copying the files on the PC and open it there, the changes are not in the shapefile and it says that the file was last edited when it was created. The changes I made while using QField are not registered., but they are still visible when I open the project in the app on the device?

Is it possible to remedy this problem by uploading the file to Qfieldcloud? Is there anything I did wrong?

Any help would be greatly appreciated, I dont want to loose all the entries I made.

Edit: PC is running on Windows 10 Enterprise, Tablet used for entering is a Galaxy Tab active 3, running Android 11.

**Comment by Thomas-2022** [comment]
Nach den aktuellen Updates werden die Änderungen an den Shapes nun im Speicherort des importierten Projektes gespeichert: 
Android/data/ch.opengis.qfield/files/
